Open Judge System/Web/OJS.Web/Areas/Administration/ViewModels/ContestCategory/ImportContestsViewModel.cs

```diff
@@ -0,0 +1,35 @@
+namespace OJS.Web.Areas.Administration.ViewModels.ContestCategory
+{
+    using System.ComponentModel.DataAnnotations;
+
+    using static OJS.Common.Constants.EditorTemplateConstants;
+
+    public class ImportContestsViewModel
+    {
+        public int CategoryId { get; set; }
+
+        public string CategoryName { get; set; }
+
+        [Display(Name = "Contest IDs to Import")]
```

Comment:
Shouldn't these strings be exported into constants?

Reply:
Done